dev/com.ibm.ws.ejbcontainer.core/resources/com/ibm/ejs/container/container.nlsprops

```diff
@@ -1566,6 +1566,14 @@ UNABLE_TO_BIND_REMOTE_BINDING_CNTR0341W=CNTR0341W: Unable to bind the {0} interf
 UNABLE_TO_BIND_REMOTE_BINDING_CNTR0341W.explanation=The remote enterprise bean binding cannot be created because the CORBA name server is not available.
 UNABLE_TO_BIND_REMOTE_BINDING_CNTR0341W.useraction=Check the log file for exceptions that are related to the ORB or CORBA name server initialization.
 
+CNTR0342E_INCORRECT_ASYNC_ANNO=CNTR0342E: The {0} bean in the {1} module of the {2} application has one or more methods with the jakarta.enterprise.concurrent.Asynchronous annotation, which must not be used on enterprise beans. The jakarta.ejb.Asynchronous annotation provides asynchronous methods for enterprise beans.
+CNTR0342E_INCORRECT_ASYNC_ANNO.explanation=The Jakarta Concurrency specification requires that the jakarta.enterprise.concurrent.Asynchronous annotation not be used on Jakarta Enterprise Beans.
+CNTR0342E_INCORRECT_ASYNC_ANNO.useraction=Remove the Jakarta Concurrency Asynchronous annotation, jakarta.enterprise.concurrent.Asynchronous, from the method or Class. Use the Jakarta Enterprise Beans Asynchronous annotation, jakarta.ejb.Asynchronous, to identify asynchronous enterprise bean methods.
+
+CNTR0343W_INCORRECT_ASYNC_ANNO_INTERFACE=CNTR0343W: The {0} business interface contains a jakarta.enterprise.concurrent.Asynchronous annotation. These annotations are not valid on interface types, and the Jakarta Enterprise Beans container ignores them.
```

Comment:
`These annotations are not valid on interface types`

This reads strangely- is the annotation not valid for _any_ interface type?

Reply:
@dmuelle I agree this statement was confusing.  I have changed it to: 
```
Asynchronous annotations are not valid on any interface, and the Jakarta Enterprise Beans container will ignore them.
```
To indicate that in addition to it being the wrong annotation, a user should know that even the EJB Asynchronous annotation is not valid on an interface. 
FYI - @tkburroughs

Reply:
👍